pkg/libvirttools/virtualization.go

```diff
@@ -91,10 +112,40 @@ func (ds *domainSettings) createDomain(config *types.VMConfig) *libvirtxml.Domai
 		emulator = noKvmEmulator
 	}
 
+	var deviceIDs []pciInfos
+
+	for _, env := range config.Environment {
+		if strings.Contains(env.Key, "QAT") {
+			devinfo := getPciInfo(env.Value)
+			deviceIDs = append(deviceIDs, devinfo)
+		}
+	}
+
+	var hostdevs []libvirtxml.DomainHostdev
+	if len(deviceIDs) > 0 {
```

Comment:
This `if` is useless. If slice will be empty - for loop body will just be not executed.

Reply:
Yeah, it is useless. Thanks